docs/ide/default-keyboard-shortcuts-in-visual-studio.md

```diff
@@ -53,6 +53,7 @@ The sections in the following table include commands that are global in that you
 
 |Commands|Keyboard shortcuts|
 |--------------| - |
+|Build.BuildSelection|**Ctrl+B**|
```

Comment:
```suggestion
|Build.BuildSelection|**Ctrl+B** or **Shift**+**F6** (Visual Studio 2019 only)|
```

Reply:
On my machine Shift-F6 is def not Build Selected.

F6 is Window.NextSplitPane and Shift-F6 is Window.PreviousSplitPane.

I can't understand why you are seeing different keys from me -mine are 100% default AKAIK

Reply:
Ahh okay, Shift+F6 must be for the C# development profile.